### DISCUSSION TITLE
Option to add more details in checkout field along with putting place holder

### DISCUSSION THREAD

**Discussion by darksrider** [woocommerce/woocommerce#52355] [Checkout Flow]
If we getting inbuilt option to add custom details like - Landmark, locality etc along with placeholder will make it easy to cutmoise

**Comment by alexflorisca** [comment]
HI @darksrider you can add additional fields to the checkout using the [additional fields API](https://developer.woocommerce.com/docs/cart-and-checkout-additional-checkout-fields/)
